tests/assets.rs

```diff
@@ -2,14 +2,14 @@
 mod tests {
 
     use async_std::task;
-    use iroha::{model::model, storage::kura};
+    use iroha::{model, storage::kura};
```

Comment:
Maybe add `model::command` to **use** otherwise a lot of code duplication.

Reply:
old code, will fix warnings in next PR